test/rest.test.js

```diff
@@ -2227,7 +2227,6 @@ describe('strong-remoting-rest', function() {
         for (var prop in expected) {
           expect(result.body.error[prop], prop).to.equal(expected[prop]);
         }
-        expect(result.body.error.stack, 'stack').to.contain(__filename);
```

Comment:
Why is this change needed? I don't mind much, I just want to ensure we fully understand why we are making it.